<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupConsole">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>801</width>
      <height>341</height>
     </rect>
    </property>
    <property name="title">
     <string>Signing in console</string>
    </property>
    <widget class="QLabel" name="lblOfLoginPicture">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>511</width>
       <height>301</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupSignIn">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>60</y>
       <width>251</width>
       <height>231</height>
      </rect>
     </property>
     <property name="title">
      <string>SignIn</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>UserName</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtUserName"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>3</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Pass word</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtPassWord"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonLogin">
        <property name="text">
         <string>Login</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>830</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
